docs/api/math/Color.html

```diff
@@ -185,6 +185,11 @@
 		Adds given h, s, and l to this color's existing h, s, and l values.
 		</div>
 
+		<h3>[method:Color sub]( [page:Color color] ) [page:Color this]</h3>
+		<div>
+		Subs rgb values of given color to rgb values of this color
```

Comment:
Subtracts rgb components of given color from rgb components of this color. If a component is negative, it is set to zero.